AspNet.Security.OAuth.Snapchat/AspNet.Security.OAuth.Snapchat.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>ASP.NET Core security middleware enabling Snapchat authentication.</Description>
+    <Authors>Nikola Maksimovic</Authors>
+    <PackageTags>aspnetcore;authentication;oauth;security;snapchat</PackageTags>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations" Version="2022.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.8" />
```

Comment:
Versions are defined centrally, not in the individual projects. We also only support .NET 6 (.NET 2.2 has been out of support for [since 2019](https://devblogs.microsoft.com/dotnet/net-core-2-2-will-reach-end-of-life-on-december-23-2019/)).

```suggestion
    <PackageReference Include="JetBrains.Annotations" />
    <PackageReference Include="Microsoft.AspNetCore.App" />
```